scheduler/src/cook/kubernetes/controller.clj

```diff
@@ -21,45 +21,66 @@
   ; TODO: Should do lock sharding based on hash of pod-name.
   lock-object)
 
-(defn expected-state-equivalent?
+(defn cook-expected-state-equivalent?
   "Is the old and new state equivalent?"
   [old-state new-state]
   (= old-state new-state); TODO
   )
 
-(defn existing-state-equivalent?
+(defn k8s-actual-state-equivalent?
   "Is the old and new state equivalent?"
   [old-state new-state]
   ; TODO:
   ; This is not right. We need to tune/tweak this to
   ; suppress otherwise identical states so we don't spam.
   (= old-state new-state))
 
-(defn remove-finalization-if-set-and-delete
-  [api-client expected-state-dict pod]
-  (api/remove-finalization-if-set-and-delete api-client pod)
-  expected-state-dict)
+(defn delete-pod
+  "Kill pod is the same as deleting a pod. I semantically distinguish them. Delete is used for completed pods that
+  we're done with. Kill is used for possibly running pods we want to kill so that they fail. Returns a new
+  cook expected state dict of nil."
+  [api-client pod]
+  (api/delete-pod api-client pod)
+  nil)
 
-(defn kill-task
-  [api-client expected-state-dict pod]
-  (api/kill-task api-client pod)
-  expected-state-dict)
+(defn kill-pod
+  "Kill pod is the same as deleting a pod. I semantically distinguish them. Delete is used for completed pods that
```

Comment:
We makes sense. I don't see a nice way to merge, and both sentences work well standalone.

Reply:
Regarding the sentences I meant something as follows where each sentence is in its own line:

```
Kill pod is the same as deleting a pod, we semantically distinguish the two operations. 
Delete is used for completed pods that we're done with. 
Kill is used for possibly running pods we want to kill so that they fail. 
Returns the cook-expected-state-dict passed in.
```

Reply:
Thanks for the text. Put in. I'll put in a nitpick that sentences on different lines is inconsistent with how we do comments elsewhere in this code and in cook. (because this strategy scales badly if you need to write a sentence more than ~150 chars)